src/Views/batchprocessor.html

```diff
@@ -15,7 +15,24 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
 
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-            <p>This is the submit batch processor tab</p>
+            <div id="selectRegionID">
+                <label>Select State / Region:</label>
+
+                <select ng-model="selectedRegion" ng-options="x.name for x in vm.regionList | orderBy: 'name'"></select>

```

Comment:
```suggestion
                <select ng-model="selectedRegion" ng-options="region.name for region in vm.regionList | orderBy: 'name'"></select>

```

Improves readability